src/isomorphic/constructRoutes.js

```diff
@@ -16,9 +16,10 @@ import { find } from "../utils/find";
 // The weird strings here are to trick rollup into not optimizing the import to avoid the * syntax
 const defaultStr = "d" + "efault";
 const pathToActiveWhenStr = "pathTo" + "ActiveWhen";
-const pathToActiveWhen = singleSpa[defaultStr]
-  ? singleSpa[defaultStr][pathToActiveWhenStr]
-  : singleSpa[pathToActiveWhenStr];
+const pathToActiveWhen =
+  typeof singleSpa[defaultStr] !== "undefined"
```

Comment:
without the `typeof` check, certain versions of webpack won't be able to use single-spa-layout.